src/components/appcontent.tsx

```diff
@@ -77,7 +79,7 @@ const AppContent: FC<Props> = ({ showSidePane, showNavbar, appContent }) => {
             </div>
           )}
           <div
-            className={`w-full ${
+            className={`sm:w-full ${
```

Comment:
Since this is used across all pages.
Did u check if this breaks other pages ?

Reply:
removed sm